examples/classification/README.md

```diff
@@ -0,0 +1,46 @@
+# Example Integration: Classification
+
+This example integration uses the [CIFAR-10](https://www.cs.toronto.edu/~kriz/cifar.html) and open-source
+classification models to demonstrate how to test multiclass classification problems on Kolena.
+
+## Setup
+
+This project uses [Poetry](https://python-poetry.org/) for packaging and Python dependency management. To get started,
+install project dependencies from [`pyproject.toml`](./pyproject.toml) by running:
+
+```shell
+poetry update && poetry install
+```
+
+## Usage
+
+The data for this example integration lives in the publicly accessible S3 bucket `s3://kolena-public-datasets`.
+
+First, ensure that the `KOLENA_TOKEN` environment variable is populated in your environment. See our
+[initialization documentation](https://docs.kolena.io/installing-kolena/#initialization) for details.
+
+### Multiclass Classification on CIFAR-10
+
+This project defines two scripts that perform the following operations:
+
+1. [`seed_test_suite.py`](scripts/multiclass/seed_test_suite.py) creates the following test suite:
```

Comment:
small nit for this (and other examples): I'd love to see a shell command that I can just copy to know that there aren't any other flags to run with, even if it's as simple as:
```
poetry run python3 scripts/multiclass/seed_test_suite.py
```

Reply:
Gotcha. Will add this in the next PR!

Reply:
Done!